tests/Doctrine/Tests/ORM/Functional/Ticket/DDC5684Test.php

```diff
@@ -66,36 +66,30 @@ public function testFetchObjectWithAutoIncrementedCustomType(): void
     }
 }
 
-class DDC5684ObjectIdType extends DBALTypes\IntegerType
+class DDC5684ObjectIdType extends DBALTypes\BigIntType
```

Comment:
This is a hack to be able to return a non-integer value from `convertToPHPValue()` and have the field autoincremented at the same time. Not sure how valid this test is in the first place.